### ISSUE TITLE
Ridiculously big icons on KDE/Plasma 5 (Kubuntu Xenial 16.04.1)

### ISSUE THREAD

**Issue by psit** [FreeRDP/Remmina#1011]
Version: 1.2.0-rcgit-15 (git n/a) (from official PPA)
OS: Kubuntu 16.04.1 Xenial
KDE Frameworks 5.18.0
Qt 5.5.1

The icons of the toolbar are ridiculously big and i can't find an option to make them smaller. See the attached screenshot - it is self explaining.
![screenshot_20161007_132608](https://cloud.githubusercontent.com/assets/4255692/19188401/c091c01e-8c91-11e6-8fb8-49cd2168ee66.png)

**Comment by antenore** [comment]
This is clearly a theme issue, that icons are not provided by remmina, only the small one it is.

**Comment by psit** [comment]
You are right. They are provided by KDE's style settings for GNOME/GTK applications. (The "symbol designs", not the gtk-designs)
But still, the "Breeze" Icons that you see in the screenshots are SVG graphics. Isn't there a way to scale them down to a proper size? 

I changed my settings to use oxygen theme now, which makes it better. Again here, there are buttons in several sizes available as PNG and they are still a bit too large for my favour. The icons are 32 pixels wide, which is ok, but the button has a padding of 8 px, so the whole button is like 48 pixels wide (which is by far better than the approx. 80x80 buttons I had!). 

Maybe all of this is related to the GTK2 settings for my KDE - I'm using KDE for many years now, but I never had such an issue with GTK applications.

**Comment by antenore** [comment]
I'll take a better look ASAP (I'm very busy now).

In the meanwhile you can take a look [here](https://github.com/FreeRDP/Remmina/tree/next/remmina/icons) and [here](https://github.com/FreeRDP/Remmina/tree/next/remmina/desktop) if you see anything wrong...

The file that sets the icons in the main window is https://github.com/FreeRDP/Remmina/blob/next/remmina/src/remmina_connection_window.c and it's the file that, in general, needs more to be refactored in Remmina, so don't worry if you'll feel lost looking at it.

Thanks for your support!!!!!

**Comment by giox069** [comment]
As workaround: set the GTK3 theme under KDE to "Adwaita".
This can be done by editing the file ~/.config/gtk-3.0/settings.ini and change the line
`gtk-icon-theme-name=XXXX`
to
`gtk-icon-theme-name=Adwaita`

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @giox069 @psit
